src/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+# Copyright 2022 Hennadii Stepanov
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or https://www.opensource.org/licenses/mit-license.php.
+
+add_definitions(-DHAVE_CONFIG_H)
+include_directories(${PROJECT_SOURCE_DIR} ${PROJECT_SOURCE_DIR}/include)
+include_directories(${PROJECT_BINARY_DIR}/src)
```

Comment:
```suggestion
```
These lines can (and should) be removed once https://github.com/bitcoin-core/secp256k1/pull/1116 has been merged.

The only paths that needs to be set then is `${PROJECT_SOURCE_DIR}/include` for the code examples.

Reply:
While the `libsecp256k1-config.h` is still present in the project, including of `PROJECT_BINARY_DIR` is required because it is the place where all build artifacts, including the `libsecp256k1-config.h`, reside.

Reply:
`include_directories(${PROJECT_SOURCE_DIR} ${PROJECT_SOURCE_DIR}/include)` has been [removed](https://github.com/bitcoin-core/secp256k1/pull/1113#issuecomment-1172718404).